common/test/acceptance/pages/lms/account_settings.py

```diff
@@ -0,0 +1,61 @@
+"""
+Base class for account settings page.
+"""
+from . import BASE_URL
+
+from bok_choy.page_object import PageObject
+from bok_choy.promise import EmptyPromise
+
+from .fields import FieldsMixin
+
+
+class AccountSettingsPage(FieldsMixin, PageObject):
+    """
+    Tests for Account Settings Page.
+    """
+
+    url = "{base}/{settings}".format(base=BASE_URL, settings='account/settings')
+
+    def is_browser_on_page(self):
+        return 'Account Settings' in self.browser.title
```

Comment:
The bok-choy visit method goes to the page then does a promise using is_browser_on_page.
The browser title fulfill this promise too quickly.
You should instead put something here that will show not only the title is correct but some actual elements on the page have been rendered. Pick something within that page and make sure it's there with something like `return self.q(css='foo').present`